### DISCUSSION TITLE
How to send any ERC20 token by useSendTransaction ? or different hook?

### DISCUSSION THREAD

**Discussion by PrzemKam001** [wevm/wagmi#3249] [Q&A]
Hello, 
Can You help me to build transaction by wagmi hook useSendTransaction but not natvie token of the chain but my own from my own contract? 

Thank You

**Comment by POKENA7** [comment]
@PrzemKam001 
Better to use `usePrepareContractWrite` and `useContractWrite` hooks.
This document is helpful.
https://wagmi.sh/examples/contract-write-dynamic
